source/lac/vector_memory_release.inst.in

```diff
@@ -0,0 +1,34 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 1999 - 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+
+for (VECTOR : SERIAL_VECTORS)
+{
+    \{
+    dealii::GrowingVectorMemory<dealii::VECTOR> memory;
+    memory.release_unused_memory();
```

Comment:
This is a static function -- you can just call it on the class, no need to create an object.